policyengine_us/variables/gov/states/mo/tax/income/deductions/mo_pension_and_ss_or_ssd_deduction_section_c.py

```diff
@@ -25,9 +25,7 @@ def formula(person, period, parameters):
         ind_taxable_ben = person("taxable_social_security", period)
         unit_taxable_ben = tax_unit.sum(ind_taxable_ben)
         unit_deduction = max_(0, unit_taxable_ben - unit_agi_over_allowance)
-        ind_frac = where(
-            ind_taxable_ben > 0,
-            ind_taxable_ben / unit_taxable_ben,
-            0,
-        )
+        ind_frac = np.zeros_like(ind_taxable_ben)
```

Comment:
```suggestion
        # Compute the individual's share of the tax unit's taxable benefits.
        # Use a mask rather than where to avoid a divide-by-zero warning. Default to zero.
        ind_frac = np.zeros_like(ind_taxable_ben)
```